SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
 [sdpi_offset=1]
 === Requirements:  From Narratives to Plug-and-Trust Interfaces
 
+For every device or system interface that supports the elements specified in this technical framework, requirements and capabilities are derived from many different standards, types of standards, and standards development organizations (SDO).  Some are technology focused, such as the ISO/IEEE 11073 standards for health device communication, and others are more focused on quality and risk management, such as the ISO/IEC 80001-1 and ISO 14971 standards.  They are all profiled and implemented and tested in a single interface point, and thus need to be integrated in a coherent and cohesive manner.
+
+In order to manage this diversity and wealth of specifications and standards, the following <<hanging_gardens_framework>> graphic was created:
+
+.Hanging Gardens Framework
+
```

Comment:
```suggestion
[#hanging_gardens_framework]
```

Reply:
Agree on the need to generally avoid "The following ...", though in this case - it is pretty obvious.  That said, how would you integrate in-line text for the autoreferences for Figure 1. Hanging Gardens Framework?  Which is defined (vs. defining a [#...] anchor.  In other words, how can I call out the text of the reference "Hanging Gardens Framework" and separately the number "Figure 1"?    

Similar to :xrefstyle" basic vs. short ... only with inline text vs. document attributes.

Reply:
No clue and it will be subject to change anyway - because of our custom numbering...

It should be sufficient to just refer to <<...>> without taking care of any details. Replacing the reference by actual text should be the job of the asciidoc compiler/converter so that each reference is transformed uniformly.